test/functional/fixtures/auth/test.js

```diff
@@ -0,0 +1,13 @@
+describe('Basic and NTLM authentications', function () {
+    it('Try basic authentication with wrong credentials', function () {
```

Comment:
I think this test should have flag `shouldFail` and we should check error message and callsite